README.md

```diff
@@ -1,10 +1,18 @@
 # Software Engineering: Homework-2b
 
-Write a "good" Github Repo
-
```

Comment:
Instead of deletion, this should be added somewhere, because it is verbatim from our HW-2b website. Other goals are description of this concise notion.